app/src/main/AndroidManifest.xml

```diff
@@ -17,7 +17,6 @@
         <activity
             android:name=".MainActivity"
             android:exported="true"
-            android:label="@string/app_name"
```

Comment:
Why is this removed?

Reply:
I think it was redundant.

Reply:
Android Studio told me it's redundant, so I removed it.